program/rust/src/validator.rs

```diff
@@ -128,3 +141,45 @@ pub fn aggregate_price(
             .to_bytes(),
     ])
 }
+
+fn checked_load_price_account(price_account_info: &[u8]) -> Option<&PriceAccount> {
+    check_price_account_header(price_account_info).ok()?;
+    Some(bytemuck::from_bytes::<PriceAccount>(
+        &price_account_info[0..size_of::<PriceAccount>()],
+    ))
+}
+
+pub fn compute_publisher_stake_caps(
```

Comment:
Please document the arguments of this function.

Reply:
Done